asg/variables.tf

```diff
@@ -117,3 +105,28 @@ variable "tags" {
   }
 }
 
+variable "block_devices" {
+  type         = list(object({
+                   device_name = string,
+                   delete_on_termination = bool,
+                   encrypted = bool,
+                   iops = number,
+                   snapshot_id = string,
+                   throughput = number,
+                   volume_size = number,
+                   volume_type = string
+                 }))
+  description = "List of block_device_mappings for the launch template. See the `block_device_mappings` block in the aws_launch_template resource for descriptions of the fields."
+  default = [
+    {
+      device_name = "/dev/xvda",
+      delete_on_termination = true,
+      encrypted = true,
```

Comment:
It looks like the old default for `volume_encryption` was `false`. I don't necessarily think it's bad to encrypt volumes by default is a bad thing, but I just wanna point out that this is a breaking change for any consumers of the module